packages/office-ui-fabric-react/src/components/Coachmark/Coachmark.tsx

```diff
@@ -47,12 +49,36 @@ export interface ICoachmarkState {
    * Is the mouse in proximity of the default target element
    */
   isMouseInProximity: boolean;
+
+  /**
+   * Containing element that the beak will use to position
+   * itself
+   */
+  targetBeakContainer?: HTMLElement;
```

Comment:
This is never used, will it ever be different than the coachmark target?

Reply:
Going to remove this.